client/src/TagExplorer/TagExplorerComponent.js

```diff
@@ -308,13 +336,22 @@ export default class TagExplorerComponent extends React.Component {
                 <KeyConceptList
                   question_answer_pairs={_.map(
                     [
-                      "MtoM_tag_warning_reporting_level",
-                      "MtoM_tag_warning_resource_splitting",
-                      "MtoM_tag_warning_double_counting",
+                      [
+                        "MtoM_tag_warning_reporting_level_q",
+                        "MtoM_tag_warning_reporting_level_a",
+                      ],
+                      [
+                        "MtoM_tag_warning_resource_splitting_q",
+                        "MtoM_tag_warning_resource_splitting_a",
+                      ],
+                      [
+                        "MtoM_tag_warning_double_counting_q",
+                        "MtoM_tag_warning_double_counting_a",
+                      ],
                     ],
-                    (key) => [
-                      <TM key={key + "_q"} k={key + "_q"} />,
-                      <TM key={key + "_a"} k={key + "_a"} />,
+                    (pair) => [
+                      <TM key={pair[0]} k={pair[0]} />,
+                      <TM key={pair[1]} k={pair[1]} />,
```

Comment:
Small note on this pattern, which shows up somewhere else in this PR too I think, you can use array destructuring (much like object destructuring) to declare array elements as named variables. Very useful in function signatures, would look like this here `([q_key, a_key]) => `. Makes it easier to write clean code with clear names in cases like this :+1: